src/resources/addys/client.ts

```diff
@@ -1,4 +1,5 @@
 import { RainbowFetchClient } from '@/rainbow-fetch';
+import { time } from '@/utils';
```

Comment:
Better to import from `@/utils/time`. Been meaning to remove the index file entry.